a10_octavia/api/drivers/driver.py

```diff
@@ -149,6 +157,12 @@ def member_update(self, old_member, new_member):
                    constants.MEMBER_UPDATES: member_dict}
         self.client.cast({}, 'update_member', **payload)
 
+    def member_batch_update(self, pool_id, members):
```

Comment:
we can use
super().member_batch_update(pool_id, members)

Reply:
- I prefer the behavior control by our-own not up stream library
- To have our-own error message